app/src/main/kotlin/cn/lyric/getter/hook/app/Gramophone.kt

```diff
@@ -0,0 +1,47 @@
+package cn.lyric.getter.hook.app
+
+
+import android.app.Notification
+import cn.lyric.getter.api.data.ExtraData
+import cn.lyric.getter.api.tools.Tools
+import cn.lyric.getter.hook.BaseHook
+import cn.lyric.getter.tool.HookTools.eventTools
+import cn.lyric.getter.tool.HookTools.isApi
+import cn.lyric.getter.tool.MeiZuNotification
+import cn.xiaowine.xkt.Tool.isNot
+import cn.xiaowine.xkt.Tool.isNotNull
+import com.github.kyuubiran.ezxhelper.ClassUtils.loadClass
+import com.github.kyuubiran.ezxhelper.EzXHelper.classLoader
+import com.github.kyuubiran.ezxhelper.HookFactory.`-Static`.createHook
+import com.github.kyuubiran.ezxhelper.finders.MethodFinder.`-Static`.methodFinder
+
+
+object Gramophone : BaseHook() {
+
+    override fun init() {
+        super.init()
+        loadClass("androidx.core.app.NotificationManagerCompat", classLoader).methodFinder().first { name == "notify" }.createHook {
```

Comment:
After recent change to make it work with media3 library, I think it will not be enough.
Please hook both the current notify()
and in addition:
```
public static void setForegroundServiceNotification(
      Service service,
      int notificationId,
      Notification notification,
      int foregroundServiceType,
      String foregroundServiceManifestType) {
```
looks like:
```
loadClass("androidx.media3.common.util", classLoader).methodFinder().first { name == "setForegroundServiceNotification" }.createHook {
            after {
                val notification = it.args[3] as Notification
[handle lyric notification]
```
Thank you